bodhi/tests/server/test_utils.py

```diff
@@ -88,6 +88,21 @@ def test_sorted_builds(self):
         assert b1 == new, b1
         assert b2 == old, b2
 
+    def test_splitter(self):
+        splitlist = util.splitter(["build-0.1", "build-0.2"])
+        assert splitlist[0] == "build-0.1"
+        assert splitlist[1] == "build-0.2"
```

Comment:
Done. I had to make the class in this file extend unittest.TestCase to be able to use assertEqual though.